src/pibasis.jl

```diff
@@ -166,6 +166,19 @@ setreal(basis::PIBasis, isreal::Bool) =
 
 maxcorrorder(basis::PIBasis) = maxcorrorder(basis.spec)
 
+getval(A2Bmap) =  [norm(A2Bmap[i,j].val) for i = 1:size(A2Bmap)[1], j = 1:size(A2Bmap)[2]]
+
+function scaling(pibasis::PIBasis, p)
```

Comment:
this implementation relies on there being just an `Rn Ylm` basis. This is not how things work anymore. The new code is much more flexible. As a first implementation we can try 
```julia 
ww[i] += sum( abs.(values(b)).^p )
```
but this is still not great tbh. so please add a line 
```julia
# TODO: revisit how this should be implemented for a general basis
```

Reply:
I revised it following your suggestions and it works for now (with the same output as the original one). Thanks for the suggestion and I will consider the `TODO` thing later.